dataclass_rest/rest.py

```diff
@@ -19,10 +19,12 @@ def rest(
         additional_params = {}
 
     def dec(func: Callable) -> Method:
+        new_url_template = (lambda *args, **kwargs: url_template) if isinstance(url_template, str) else url_template
+
         method_spec = parse_func(
             func=func,
             body_param_name=body_name,
-            url_template=url_template,
+            url_template=new_url_template,
```

Comment:
let's store original string as well. It call be helpful for debugging

Reply:
how can i store it? pass it as different argument?